.circleci/config.yml

```diff
@@ -118,21 +118,33 @@ jobs:
           paths:
             - target/uberjar/rems.jar
 
-  rahti-dev:
+  docker-circle:
```

Comment:
would `docker-snapshot` be a better name for this?

Reply:
Sure